### DISCUSSION TITLE
Multi-option tracker (for Mood tracking)

### DISCUSSION THREAD

**Discussion by passthesh3ll** [iSoron/uhabits#979] [Feature Requests]
Hi, I'd like to have a new type of tracking "object" in the app (other than "numerical tracker" and "yes/no tracker"): a multi option tracker. It can be very useful for example to track mood (happy, angry, sad..), with a different color for each value. It would be very useful if the tracker line continues to have a color (in order to visually group multiple lines by color on the left), and inside that, each value has a different color.
_Example of mood tracking_
![asd](https://user-images.githubusercontent.com/63223659/121854557-9f32f180-ccf2-11eb-9ff3-9591e473124e.png)

It should be useful not only for mood tracking but also for example for different variants of the same task. If I want to track meditation or sleep with Multi-option tracking, I have inside one line two information: I meditated + I have meditated with guide/audioguide/alone/etc..
_Example of meditation tracking_
![asd](https://user-images.githubusercontent.com/63223659/121855369-8bd45600-ccf3-11eb-8feb-10bb605d89e5.png)

**Comment by ke352802081770314** [comment]
While I see this sort of thing may be useful, I don't really feel it meshes well with Loop Habit Tracker. This app is about forming habits and assessing the strength of routine/compiance already achieved. Multiple-choice have no meaning to the app, so provide no basis to evaluate them against a scale or derive compliance/achievement from them. Unless you had an additional configuration where you can assign values to the choices (e.g. happy=90 angry=30 sad=10 or whatever). So it doesn't fit into the scheme and if any statistics can be derived, they would have to be different from the ones already provided, meaning a lot of effort and added complexity of the app and its appearance. (Note I am only commenting my personal opinion as a user of the app)
The meditation example, which basically means you have multiple values of "yes", is somewhat similar to the existing feature request to be able to annotate an entered value with a comment, and could be used as an improvement to that by specifying a visualization.

**Comment by auipga** [comment]
I like this idea.
During the number input there could be a list of tags or names, that are preset with values. This will lower the time to input 90 (happy) and 30 (angry) if one could just tap that preset. We could also configure to not show the value field at all and only show the tags (each representing its value). The overview could also show the tags first letter (or few specified letters with specidied color by that tag).
If each type of answer (happy, sad,... / guide,audioguide,alone,...) has its internally configured value, there will be no change for the stats and calculation at all.
